src/modules/Dropdown/Dropdown.js

```diff
@@ -949,6 +962,17 @@ export default class Dropdown extends Component {
   }
 
   // ----------------------------------------
+  // Overrides
+  // ----------------------------------------
+
+  handleIconOverrides = predefinedProps => ({
+    onClick: (e) => {
+      _.invoke(predefinedProps, 'onClick', e, predefinedProps)
```

Comment:
Please take an attention to this line, `Icon` component doesn't have `onClick` handler, so it doesn't have second argument. There are two possible solutions: remove the second argument or add `onClick` hanlder to the `Icon` component.